pkg/component/component.go

```diff
@@ -68,6 +68,31 @@ func CreateFromGit(client *occlient.Client, name string, ctype string, url strin
 	return nil
 }
 
+// GetComponentPorts provides slice of ports used by the component in the form port_no/protocol
+func GetComponentPorts(client *occlient.Client, componentName string, applicationName string) (res []string, err error) {
```

Comment:
a different variable name than `res` would be better.

Maybe `ports` ?

Reply:
Ack